### PR TITLE
Update instance segmentation demo

### PR DESCRIPTION
**Pull request by druzhkov-paul**
This PR updates Python instance segmentation demo and introduces the following changes:

- Readme fixes
- Command line parameters' alignment with other demo applications
- Support of alternative image resize mode

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
@vmatveic @snosov1@DmitriySidnev please take a look

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vmatveic** [comment]
Looks good to me but I didn't check the main logic deeply. I've just briefly checked that all the required topics (from the main issue) are covered. @snosov1 , @Wovchena , could you please check the logic or assign someone else?
@druzhkov-paul , did you ask tech writers to review documentation? @azaytsev1980 @aalborov could you please check?
Thanks!

File: demos/python_demos/instance_segmentation_demo/README.md [hunk 1]
```diff
@@ -73,7 +80,13 @@ To run the demo, you can use public or pre-trained models. To download the pre-t
 
 To run the demo, please provide paths to the model in the IR format, to a file with class labels, and to an input video, image, or folder with images:
 ```bash
-python3 instance_segmentation_demo/main.py -m <path_to_model>/maskrcnn_r50_fpn_2x.xml --label instance_segmentation_demo/coco_labels.txt --video 0
+python3 instance_segmentation_demo/instance_segmentation_demo.py \
+    -m <path_to_model>/instance-segmentation-security-0050.xml \
+    --label instance_segmentation_demo/coco_labels.txt \
+    -l <openvino_root>/inference_engine/lib/intel64/libcpu_extension_avx2.so \
+    --no_keep_aspect_ratio \
+    --video 0 \
```
Comment (Wovchena) [inline demos/python_demos/instance_segmentation_demo/README.md]: I think it must be `-i` key

Comment (druzhkov-paul) [reply demos/python_demos/instance_segmentation_demo/README.md]: Fixed.

**Comment by druzhkov-paul** [comment]
@azaytsev1980 @aalborov please review.

**Comment by IRDonch** [comment]
Jenkins: ok to test

File: demos/python_demos/instance_segmentation_demo/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Instance Segmentation Python* Demo
 
-This demo demonstrates how to run Instance Segmentation models from `Detectron` or `maskrcnn-benchmark` using OpenVINO&trade;.
+This demo shows how to run Instance Segmentation models from `Detectron` or `maskrcnn-benchmark` using OpenVINO&trade;.
```
Comment (aalborov) [inline demos/python_demos/instance_segmentation_demo/README.md]: `Instance Segmentation models` -->`Instance-Segmentation models`

File: demos/python_demos/instance_segmentation_demo/README.md [hunk 3]
```diff
@@ -57,12 +57,19 @@ Options:
   -l "<absolute_path>", --cpu_extension "<absolute_path>"
                         Required for CPU custom layers. Absolute path to a
                         shared library with the kernels implementation.
+  --delay "<num>"       Optional. Interval in millisecond to wait for key
```
Comment (aalborov) [inline demos/python_demos/instance_segmentation_demo/README.md]: `Interval in millisecond to wait for key` --> `Interval in milliseconds of waiting for a key to be`

File: demos/python_demos/instance_segmentation_demo/README.md [hunk 4]
```diff
@@ -9,23 +9,23 @@ This demo demonstrates how to run Instance Segmentation models from `Detectron`
...
 2. The `im_info` input blob passes resulting resolution and scale of a pre-processed image to the network to perform inference.
 3. The demo visualizes the resulting instance segmentation masks. Certain command-line options affect the visualization:
     * If you specify `--show_boxes` and `--show_scores` arguments, bounding boxes and confidence scores are also shown.
-    * If you specify video as a source of images with the `--video` option, the demo shows the same object instance with the same color throughout the whole video using simple tracking. It assumes more or less static scene with instances in two frames being a part of the same track if intersection over union of the masks is greater than the 0.5 threshold.
+    * By default, tracking is used to show object instance with the same color throughout the whole video.
+      It assumes more or less static scene with instances in two frames being a part of the same track if intersection over union of the masks is greater than the 0.5 threshold. To disable tracking, specify `--no_track` argument.
```
Comment (aalborov) [inline demos/python_demos/instance_segmentation_demo/README.md]: `` specify `--no_track` argument.` `--> `` specify the `--no_track` argument.``

**Comment by kchechil** [comment]
@druzhkov-paul could you please comment the changes requested and fix them if they are valid?

File: demos/python_demos/instance_segmentation_demo/README.md (hunk 3 above)
Comment (druzhkov-paul) [reply demos/python_demos/instance_segmentation_demo/README.md]: Fixed. Thanks.

File: demos/python_demos/instance_segmentation_demo/README.md (hunk 4 above)
Comment (druzhkov-paul) [reply demos/python_demos/instance_segmentation_demo/README.md]: Fixed.

File: demos/python_demos/instance_segmentation_demo/README.md (hunk 2 above)
Comment (druzhkov-paul) [reply demos/python_demos/instance_segmentation_demo/README.md]: Disagree. It's "Instance Segmentation".

**Comment by druzhkov-paul** [comment]
@kchechil Done. What're the further steps?

**Comment by kchechil** [comment]
@aalborov please finish your review. @IRDonch please take a look and merge if no comments.

**Comment by aalborov** [comment]
Looks good to me.

**Pull request merged**